src/router/index.ts

```diff
@@ -32,4 +33,16 @@ const keys = createRoute({
   component: defineAsyncComponent(() => import('../views/SettingsKeysView.vue'))
 })
 
-export const router = createRouter([home, settings, profile, keys])
\ No newline at end of file
+const requiresAuth = createRoute({
+  name: 'auth',
+  path: '/requires-auth',
+  onBeforeRouteEnter: (_to, { reject }) => {
+    // @ts-expect-error
+    reject('NotAuthorized')
+  },
```

Comment:
why does this error? Aren't we registering the router in this project?

Reply:
this is where I discovered [this bug](https://github.com/kitbagjs/router/pull/470). Will bump versions and then I can remove the ts-expect-error